appinfo/routes.php

```diff
@@ -514,6 +514,27 @@
 			],
 		],
 
+		/**
+		 * Federation
+		 */
+
+		[
+			'name' => 'Federation#acceptShare',
+			'url' => 'api/{apiVersion}/federation/pending/{id}',
```

Comment:
Since database and Model/Mapper are Invitation, let's use that in the URL too:

```suggestion
			'url' => 'api/{apiVersion}/federation/invitation/{id}',
```